app/src/main/java/com/android/sample/ui/profile/ProfileCreation.kt

```diff
@@ -134,88 +148,116 @@ fun ProfileCreationScreen(
           Text(
               text = "Complete your profile creation",
               fontSize = TITLE_FONTSIZE.sp,
-              fontWeight = FontWeight.Bold, // Set the text to be bold
               modifier =
                   Modifier.padding(MEDIUM_PADDING.dp)
                       .wrapContentWidth(Alignment.CenterHorizontally)
                       .testTag("profileCreationTitle"))
-
+          Spacer(modifier = Modifier.padding(MEDIUM_PADDING.dp))
           ProfileImage(
               userId = uid,
-              modifier = Modifier.size(IMAGE_SIZE.dp).clip(CircleShape).testTag("profilePicture"),
+              modifier =
+                  Modifier.size((1.5 * IMAGE_SIZE).dp).clip(CircleShape).testTag("profilePicture"),
               imageViewModel)
 
-          Button(
-              onClick = { showDialogImage = true }, modifier = Modifier.testTag("uploadPicture")) {
-                Text("Modify Profile Picture")
+          Box(
+              modifier =
+                  Modifier.testTag("uploadPicture")
+                      .clickable { showDialogImage = true } // Handle click action
+                      .padding(MEDIUM_PADDING.dp)
+                      .background(Color.Transparent)) {
+                Icon(
+                    imageVector = Icons.Default.AddAPhoto,
+                    contentDescription = "Add a photo",
+                    tint = Color.Black)
               }
```

Comment:
I think that this part can be modularized in a mthod so it will also possible to call it from editProfile for example